Ghost protocol upgrade files/tool-orchestration-auth.ts

```
@@ -0,0 +1,520 @@
+    });
+
+    if (!response.ok) throw new Error(`Zapier webhook error: ${response.statusText}`);
+    return await response.json();
+  }
+
+  private async executeDatabaseTool(action: string, params: any, tenant: any, namespace: string) {
+    switch (action) {
+      case 'search_memories':
+        const { data: embeddingData } = await this.supabase.functions.invoke('generate-embedding', {
+          body: { text: params.query }
+        });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)